webapp/portlet/src/main/webapp/vue-apps/activity-reactions/components/ActivityReactions.vue

```diff
@@ -107,7 +107,7 @@ export default {
   },
   methods: {
     openDrawer() {
-      this.$root.$emit(`open-reaction-drawer-selected-tab-${this.activityId}`, this.activityId,'like');
+      this.$root.$emit(`open-reaction-drawer-selected-tab-${this.activityId}`, this.activityId,'like','ACTIVITY');
```

Comment:
You can send them in a single object.
```suggestion
      this.$root.$emit(`open-reaction-drawer-selected-tab-${this.activityId}`, this.activityId, 'like', 'ACTIVITY');
```